fpm/.gitignore

```diff
@@ -1,2 +1,2 @@
 build/*
-*/FODDER/*
+FODDER/*
```

Comment:
Please, don't commit personal `.gitignore` files to the repository, use your personal `.gitignore` instead:
```
git config --global core.excludesFile '~/.gitignore'
echo 'FODDER/*' >> ~/.gitignore
```